packages/manager/cypress/e2e/core/linodes/create-linode-in-core-region.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { Region } from '@linode/api-v4';
+import { linodeFactory, regionFactory } from 'src/factories';
+import { mockAppendFeatureFlags } from 'support/intercepts/feature-flags';
+import { mockCreateLinode } from 'support/intercepts/linodes';
+import {
+  mockGetRegionAvailability,
+  mockGetRegions,
+} from 'support/intercepts/regions';
+import { ui } from 'support/ui';
+import { linodeCreatePage } from 'support/ui/pages';
+import { randomLabel, randomString } from 'support/util/random';
+import { extendRegion } from 'support/util/regions';
+
+describe('Create Linode in a Core Region', () => {
+  /*
+   * - Confirms Linode create flow can be completed with a core region
+   * - Creates a basic Nanode and confirms interactions succeed and outgoing request contains expected data.
+   */
+  it('should be able to select a core region', () => {
+    // create mocks
+    const mockRegionOptions: Partial<Region> = {
+      capabilities: ['Linodes', 'Distributed Plans'],
+      site_type: 'core',
+    };
+    const mockRegion = extendRegion(regionFactory.build(mockRegionOptions));
```

Comment:
Is `extendRegion` necessary here? I think we should prefer pure types unless absolutely necessary


Can we clean this up to just be something like:
```suggestion
    const mockRegion = regionFactory.build({
      capabilities: ['Linodes', 'Distributed Plans'],
      site_type: 'core',
    })
```